src/components/CastList.js

```diff
@@ -1,33 +1,31 @@
 import styled from 'styled-components';
-import PosterActor from './PosterActor';
 import defaultPoster from '../assets/images/profile.png';
+import PosterActor from './PosterActor';
 
 export default function CastList({ castList, listName }) {
   return (
     <>
-      <ListHeader>{listName}</ListHeader>
+      <ListHeader>{listName ? listName : ''}</ListHeader>
```

Comment:
For accessibility reasons I don't think it is a good idea to leave an empty string within your `h3`. Headings are landmarks and if there is an empty landmark, it doesn't really make sense. I think it would be better to either render the `h3` or not at all:
```suggestion
      {listName && <ListHeader>{listName}</ListHeader>}
```